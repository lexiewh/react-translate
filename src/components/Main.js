import { Container, Heading } from '@chakra-ui/react'

function Main(){
    return(
        <Container centerContent className='main-container'>
            <Heading as='h2' size='2xl'>Translate</Heading>
        </Container>
    )
}

export default Main