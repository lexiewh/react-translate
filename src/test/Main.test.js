import Main from '../components/Main';
import Enzyme, { shallow } from 'enzyme'
import toJson from 'enzyme-to-json'
import Adapter from 'enzyme-adapter-react-16'

Enzyme.configure({ adapter: new Adapter() })

it('correctly compares to the snapshot', () => {
  const wrapper = shallow(<Main />)

  expect(toJson(wrapper)).toMatchSnapshot()
})