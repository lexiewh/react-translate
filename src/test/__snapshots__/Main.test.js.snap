// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`correctly compares to the snapshot 1`] = `
<Container
  centerContent={true}
  className="main-container"
>
  <Heading
    as="h2"
    size="2xl"
  >
    Translate
  </Heading>
</Container>
`;
